charts/sourcegraph/templates/qdrant/qdrant.ConfigMap.yaml

```diff
@@ -0,0 +1,34 @@
+{{- if .Values.qdrant.enabled -}}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Values.qdrant.name }}
+  labels:
+    deploy: sourcegraph
+    app.kubernetes.io/component: qdrant
+data:
+  config.yaml: |
+    debug: true
+    log_level: INFO
+    storage:
+      storage_path: /data
+      snapshots_path: /data/storage
+      on_disk_payload: true
+    performance:
+      max_optimization_threads: 4
+    optimizers:
+      max_optimization_threads: 4
+      mmap_threshold_kb: 1
+      indexing_threshold_kb: 0
+    hnsw_index:
+      m: 8
+      ef_construct: 100
+      full_scan_threshold: 10
+      max_indexing_threads: 4
+      on_disk: true
+      payload_m: 8
```

Comment:
Will probably pull these out to make them configurable, but they are just default values that can be overridden on collection creation which is handled in the application.

Reply:
let's add a note in the manifest and say they're basically unused